tools/osbuilder/node-builder/azure-linux/Makefile

```diff
@@ -4,6 +4,7 @@
 #
 BUILD_TYPE := release
 
+export OS_VERSION ?= $(shell sort -r /etc/*-release | gawk 'match($$0, /^(VERSION_ID=(.*))$$/, a) { print toupper(a[2] a[3]); exit }' | tr -d '"')
```

Comment:
Hmm okay am I understanding this right:

* Pro: If I build on AzL, my workflow is unchanged - I don't need to specifiy OS_VERSION because it'll be detected automatically.
* Con: If I build on Ubuntu (is this even supported), this autodetection would not work?

Reply:
This is the 'azure-linux' node builder :)

Reply:
Which is only expected to be used from AzL?